src/components/app-content-switcher/AppContentSwitcher.tsx

```diff
@@ -1,12 +1,36 @@
+import { TypographyProps } from '@mui/material/Typography'
 import { Typography } from '@mui/material'
 import Tooltip from '@mui/material/Tooltip'
 import Switch from '@mui/material/Switch'
 import Stack from '@mui/material/Stack'
 import { defaultStyles } from '~/components/app-content-switcher/AppContentSwitcher.styles'
 
-const AppContentSwitcher = ({ active, handleChange, switchOptions, typographyVariant, styles }) => {
 
-  const renderBlock = (options, active) => (
+interface SwitcherProps {
+    active: boolean;
+    handleChange: () => void;
+    switchOptions: {
+        left?: {
+            text: string;
+            tooltip?: string;
+        };
+        right?: {
+            text: string;
+            tooltip?: string;
+        };
+    };
+    typographyVariant: TypographyProps['variant'];
+    styles?: object;
+}
+
+type SwitchOption = {
```

Comment:
you allready have this type, so you can use it in the interface above

Reply:
refactored